locale/fi/toast.json

```diff
@@ -0,0 +1,50 @@
+{
+  "title": "Ponnahdusilmoitus (Toast)",
+  "intro": "Ponnahdusilmoitus on komponentti, jonka avulla käyttäjälle voidaan välittää käyttäjän toimintaan liittyvää palautetta. Käytetään tilanteissa, jotka eivät vaadi käyttäjältä varsinaista reagointia. Ponnahdusilmoitus kertoo, että käyttäjän suorittama toimenpide onnistui.",
+  "note.title": "Saavutettavuus ja käytettävyys",
+  "note.items": [
+    "Mikäli ponnahdusilmoituksia on tarkoitus lisätä sivulle dynaamisesti, niitä varten kannattaa luoda ruudunlukijakäyttäjiä varten live region(ulkoinen linkki sivun alaraunasssa), jonka sisään ponnahdusilmoitukset lisätään. Tällöin myös ruudunlukijakäyttäjät saavat tiedon sivulle lisätystä ponnahdusilmoituksesta.",
+    "Ponnahdusilmoituksen otsikkotaso on määriteltävissä käyttötapauksen mukaan.",
```

Comment:
What are the different use cases that require different heading levels? Is this relevant information? In the ingress it is mentioned that the toast is specifically for success messages, so what would the different use cases be? It's a bit confusing.

Reply:
Fixed it to be: "Ponnahdusilmoituksen otsikkotaso on määriteltävissä tarvittaessa, ja komponenttia voidaan käyttää myös ilman otsikkoa."